content/types/_realtime_client_options.textile

```diff
@@ -10,7 +10,7 @@
 
 - <div lang="java,ruby,nodejs,objc,swift,csharp"><span lang="default">recover</span><span lang="csharp">Recover</span><span lang="ruby">:recover</span></div> := This option allows a connection to inherit the state of a previous connection that may have existed under a different instance of the library by providing that connection's <span lang="default">"@recoveryKey@":/realtime/connection#recovery-key</span><span lang="ruby">"@recovery_key@":/realtime/connection#recovery-key</span>. This might typically be used by clients of an app to ensure connection state can be preserved following a reload. See "connection state recovery":/realtime/connection/#connection-state-recovery for further information and example code<br>__Type: @String@__
 
-- <span lang="default">queryTime</span><span lang="csharp">QueryTime</span><span lang="ruby">:query_time</span> := _false_ If true, the library will query the Ably servers for the current time instead of relying on a locally-available time of day. Knowing the time accurately is needed to create valid signed token requests, so this option is useful for library instances on auth servers if the server clock is untrustworthy<br>__Type: @Boolean@__
+- <span lang="default">queryTime</span><span lang="csharp">QueryTime</span><span lang="ruby">:query_time</span> := _false_ If true, the library will query the Ably servers for the current time instead of relying on a locally-available time of day. Knowing the time accurately is needed to create valid signed token requests, so this option is useful for library instances on auth servers if the server clock is untrustworthy. The server is queried only once per client library instance, and used to calculate and store the offset from the local clock.<br>__Type: @Boolean@__
```

Comment:
I think "untrustworthy" gives the wrong implication. There isn't a security risk arising from an inaccurate clock; it is just that a token request will be rejected. So I think it would be more appropriate to say the server clock is insufficiently accurate.